src/Transport/RestTransport.php

```diff
@@ -31,6 +31,7 @@
  */
 namespace Google\ApiCore\Transport;
 
+use BadMethodCallException;
```

Comment:
Hmm, this import is not used (the reference to `BadMethodCallException` below is still absolute)